extensions/src/ACRE2Core/FilterPosition.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <cmath>
 #pragma comment(lib, "x3daudio.lib")
+#pragma comment(lib, "xaudio2.lib")
```

Comment:
not fully sure why this is now required, it's unrelated to pr but need to build on VS17.6

Reply:
Interesting.

Reply:
Didn't need that on VS17.5. :thinking:

Reply:
It also builds with it for me, forgot to say that. So I am fine with leaving it in.